ext/LegendSpecFitsRecipesBaseExt.jl

```diff
@@ -490,6 +490,41 @@ end
     end
 end
 
+@recipe function f(report::NamedTuple{(:h_calsimple, :h_uncal, :c, :peak_guess, :peakhists, :peakstats)}; cal=true)
```

Comment:
This is _almost_ a duplicate of the existing recipe. However, the old one has some stuff hardcoded, e.g. "FEP" in the plot label. I didn't want to mess with the existing one, since we're int he middle of the processing.